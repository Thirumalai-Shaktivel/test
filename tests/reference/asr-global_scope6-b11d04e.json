{
    "basename": "asr-global_scope6-b11d04e",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/global_scope6.f90",
    "infile_hash": "3f2dfa122b78175565b7baa8f4d0417cfb0ed5bbe26bbb5f0f1f6f2e",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-global_scope6-b11d04e.stdout",
    "stdout_hash": "9ccd9d08b83e097778d17505a12192ac9c1dc9b5a1baff859bed3b35",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}