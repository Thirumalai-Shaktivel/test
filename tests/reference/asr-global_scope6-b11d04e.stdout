(TranslationUnit (SymbolTable 1 {x: (Variable 1 x Local () Default (Integer 4 []))}) [(= (Var 1 x) (ConstantInteger 6 (Integer 4 []))) (BinOp (ConstantInteger 2 (Integer 4 [])) Mul (Var 1 x) (Integer 4 [])) (= (Var 1 x) (BinOp (Var 1 x) Add (ConstantInteger 1 (Integer 4 [])) (Integer 4 [])))])
